packages/orbit-components/src/Loading/README.md

```diff
@@ -16,15 +16,17 @@ After adding import into your project you can use it simply like:
 
 Table below contains all types of the props available for icons in general.
 
-| Name       | Type            | Default | Description                                                                                                                                |
-| :--------- | :-------------- | :------ | :----------------------------------------------------------------------------------------------------------------------------------------- |
-| children   | `React.Node`    |         | The content of the Loading. [See Functional specs](#functional-specs)                                                                      |
-| dataTest   | `string`        |         | Optional prop for testing purposes.                                                                                                        |
-| id         | `string`        |         | Set `id` for `Loading`                                                                                                                     |
-| loading    | `boolean`       | `false` | If `true`, the Loading will be displayed. Loading which doesn't have a children is always shown, even if `loading` prop is set to `false`. |
-| **type**   | [`enum`](#enum) |         | The type of the Loading.                                                                                                                   |
-| text       | `Translation`   |         | The text of the Loading.                                                                                                                   |
-| customSize | `number`        |         | Allows you to define custom size for circle loader                                                                                         |
+| Name        | Type                      | Default | Description                                                                                                                                |
+| :---------- | :------------------------ | :------ | :----------------------------------------------------------------------------------------------------------------------------------------- |
+| children    | `React.Node`              |         | The content of the Loading. [See Functional specs](#functional-specs)                                                                      |
```

Comment:
There is no Functional Specs in this readme 🤦‍♂️. And I am very confused about what "The content of the Loading" means 😞.

This is actually the content to be shown when `loading={false}`.